services/table-analyzer.js

```diff
@@ -23,26 +22,53 @@ function TableAnalyzer(queryInterface, config) {
       case 'sqlite':
         query = `PRAGMA foreign_key_list('${table}');`;
         break;
+      default:
+        break;
     }
 
     return queryInterface.sequelize
       .query(query, { type: queryInterface.sequelize.QueryTypes.SELECT });
   }
 
-  function getType(type) {
+  function checkIfColumnIsTypeEnum(columnName) {
+    const query = `
+      SELECT i.udt_name
+      FROM pg_catalog.pg_type t
+      JOIN pg_catalog.pg_namespace n ON n.oid = t.typnamespace
+      JOIN pg_catalog.pg_enum e ON t.oid = e.enumtypid
+      JOIN information_schema.columns i ON t.typname = i.udt_name
+      WHERE i.column_name = '${columnName}'
+      GROUP BY i.udt_name;
+    `;
```

Comment:
This function seems to only works for postgres? There's no context using `queryInterface.sequelize.options.dialect`. Is it normal?

Reply:
Done

Reply:
Done? Don't see the change?

Reply:
The check is done before, since we only want to check if it is an enum when we are using postgres. https://github.com/ForestAdmin/lumber/pull/145/files#diff-3f431298269ee4103c03ca56c7f393a4R66